nxdrive/commandline.py

```diff
@@ -747,14 +752,28 @@ def __enter__(self) -> "HealthCheck":
 
         return self
 
-    def __exit__(self, *_: Any) -> None:
+    def __exit__(
```

Comment:
All changes here are not needed IMO. When a crash happens, that method will never be taken. The only way to record the stacktrace is in: https://github.com/nuxeo/nuxeo-drive/blob/6ff14d34c07b7edcc855b30fa3b957ebb7f352a6/nxdrive/fatal_error.py#L260-L265
You should use that and save `full_error` into the crash file.